contrib/scripts/run-gaia-v7.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+NODE_HOME=$(realpath ./build/.gaia)
+echo "NODE_HOME = ${NODE_HOME}"
+BINARY=$NODE_HOME/cosmovisor/genesis/bin/gaiad
+echo "BINARY = ${BINARY}"
+CHAINID=cosmoshub-4
+
+USER_MNEMONIC="abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon art"
+
+if ! test -f "./build/gaiad7"; then
+  echo "has not gaiad7"
+  exit
+fi
+
+
+rm -rf ./build/.gaia
+
+mkdir -p $NODE_HOME/cosmovisor/genesis/bin
+cp ./build/gaiad7 $NODE_HOME/cosmovisor/genesis/bin/gaiad
+$BINARY init upgrader --chain-id $CHAINID --home $NODE_HOME
```

Comment:
double quote to prevent globbing and word splitting

```suggestion
mkdir -p "$NODE_HOME"/cosmovisor/genesis/bin
cp ./build/gaiad7 "$NODE_HOME"/cosmovisor/genesis/bin/gaiad
$BINARY init upgrader --chain-id $CHAINID --home "$NODE_HOME"
```